src/test/java/com/wikia/webdriver/testcases/discussions/CreatingPostTests.java

```diff
@@ -240,6 +250,23 @@ private CategoryPill fillPostCategoryWith(final PostsCreator postsCreator, final
     return categoryPill;
   }
 
+  private String addLinkToDescription(final PostsCreator postsCreator, final String description) throws MalformedURLException {
+    final String url = "http://www.wikia.com/";
+
+    String text = description + " http://www.wikia.com/";
+    postsCreator.addDescriptionWith(new URL(url));
+
+    return text;
+  }
+
+  private PostEntity submitAndWaitForPostToAppear(final PostsCreator postsCreator, final PostsListPage page, final String description) {
+    postsCreator.clickSubmitButton();
+
+    return page.getPost()
+        .waitForPostToAppearWith(description)
+        .findNewestPost();
```

Comment:
no need to break the lines: `return page.getPost().waitForPostToAppearWith(description).findNewestPost();` has 77 characters in total